tests/test_data_iterator.py

```diff
@@ -0,0 +1,110 @@
+from re import L
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+import pandas.testing as pd_test
+import pytest
+
+from packages.python.accern_data import create_data_client, DT_FORMAT
+from packages.python.accern_data.util import (
+    DEFAULT_CHUNK_SIZE,
+    EXAMPLE_URL,
+    load_json,
+)
+
+
+@pytest.mark.parametrize("chunk_size", [1, 5, 1000, None])
```

Comment:
and one case for same size as the internal chunk_size

Reply:
no, None is to use the *default* value. I'm talking about a chunk_size that is defined and exactly the same as the chunk size as the api returns it